src/snapred/ui/view/DiffCalTweakPeakView.py

```diff
@@ -56,8 +57,10 @@ def __init__(self, jsonForm, samples=[], groups=[], parent=None):
         # create the run number field and lite mode toggle
         self.runNumberField = self._labeledField("Run Number")
         self.litemodeToggle = self._labeledField("Lite Mode", Toggle(parent=self, state=True))
+        self.maxChiSqField = self._labeledField("Max Chi Sq", QLineEdit(str(self.MAX_CHI_SQ)))
```

Comment:
I believe this value is set to a constant equal to the default in the application.yml.  Is that okay here?

Reply:
That's supposed to pre-populate the text field with the default value from within application.yml. (100.0) So it's just a place holder but can be modified by the user.